code/controllers/subsystem/traumas.dm

```diff
@@ -135,10 +136,15 @@ SUBSYSTEM_DEF(traumas)
 						/obj/item/clothing/suit/chickensuit, /obj/item/clothing/head/chicken,
 						/obj/item/clothing/suit/toggle/owlwings, /obj/item/clothing/under/owl, /obj/item/clothing/mask/gas/owl_mask,
 						/obj/item/clothing/under/griffin, /obj/item/clothing/shoes/griffin, /obj/item/clothing/head/griffin,
-						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom))
-					   )
-
-	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
+						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom)),
+						
+					   "anime" = typecahceof(list(/obj/item/clothing/under/schoolgirl, /obj/item/clothing/under/schoolgirl/red,
```

Comment:
You only need /obj/item/clothing/under/schoolgirl, the specific colours will be included with it

Reply:
sweet thanks!